database.sql

```diff
@@ -1,2 +1,8 @@
-CREATE DATABASE IF NOT EXISTS groupc_kainos_recruitment;
-USE groupc_kainos_recruitment;
+CREATE DATABASE IF NOT EXISTS groupc_kainos_recruitment_JenniferC;
+USE groupc_kainos_recruitment_JenniferC;
+
+CREATE TABLE IF NOT EXISTS job_roles (
+id smallint PRIMARY KEY AUTO_INCREMENT,
+name varchar(50),
+job_description varchar(500)
+);
```

Comment:
Might be worth adding some data inserts into the script? That way running it will have your db prepopulated